packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -539,6 +540,7 @@ class IsolateManager extends Disposer {
   Stream<IsolateRef> get onIsolateExited => _isolateExitedController.stream;
 
   final _mainIsolate = ValueNotifier<IsolateRef>(null);
```

Comment:
rather than adding a line break can you move the getter before the private field? That is actually the convention Flutter uses.